Projects/Governance/Plugins/User-Profiles/maxah.json

```diff
@@ -26,12 +26,12 @@
                     "id": "5458486a-3f31-4433-87e8-0e7b17391ba3",
                     "savedPayload": {
```

Comment:
You shouldn`t submit you gov profile with docs related stuff.

Reply:
I deleted the user profile file.